src/state/trajectory/reducer.ts

```diff
@@ -97,6 +99,20 @@ const actionToConfigMap: TypeToDescriptionMap = {
             },
         }),
     },
+    [SET_CONVERSION_ENGINE]: {
+        accepts: (action: AnyAction): action is SetConversionEngineAction =>
+            action.type === SET_CONVERSION_ENGINE,
+        perform: (
+            state: TrajectoryStateBranch,
+            action: SetConversionEngineAction
+        ) => ({
+            ...state,
+            processingData: {
+                ...state.processingData,
+                engineType: action.payload,
+            },
+        }),
+    },
```

Comment:
slightly out of scope for this PR, but we can flesh it out adjust later when we work with the engines?